### PR TITLE
Use ARM in Radix setups

### PR DESCRIPTION
**Pull request by achaikou**
I will run additional comprehensive tests on test-large once it is merged.
Tests on playground (with less powerful machines) showed that performance difference is inside error margin.

I am kinda worried that ARM actually might have worse performance in our case, but can't find anything conclusive in the web.

Also an option is to switch only test-large environment to arm, but I am not fond of having production and test-large one to be different.

Github doesn't provide standard arm machines yet, so we can't switch tests to them.

### CODE REVIEW

File: radixconfig_playground.yaml [hunk 1]
```diff
@@ -33,6 +37,7 @@ spec:
             ONESEISMIC_API_TRUSTED_PROXIES: ""
             ONESEISMIC_API_BLOCKED_IPS: ""
             ONESEISMIC_API_BLOCKED_USER_AGENTS: ""
+            OPENVDS_AZURE_IOMANAGER_THREAD_COUNT: ""
```
Comment (ajaust) [inline radixconfig_playground.yaml:40]: Is this change ARM specific?

File: radixconfig_playground.yaml [hunk 2]
```diff
@@ -43,8 +48,8 @@ spec:
           monitoring: true
           resources:
             requests:
-              memory: "500Mi"
-              cpu: "200m"
+              memory: "8192Mi"
+              cpu: "2000m"
```
Comment (ajaust) [inline radixconfig_playground.yaml:52]: Why do we request mpre more memory and CPU than x86 on playground, but not on production? Is this due to the (large) tests?

File: Dockerfile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ARG OPENVDS_IMAGE=openvds
-ARG ONESEISMIC_BASEIMAGE=golang:1.22-alpine
+ARG ONESEISMIC_BASEIMAGE=docker.io/library/golang:1.22-alpine
```
Comment (ajaust) [inline Dockerfile:2]: Why is this change needed? Does docker pull the image from another registry if we do not specify `docker.io/library`?

**Review by ajaust — APPROVED** [review]
Looks good overall. I left some small comments so I better understand what is going on. 🙂

File: radixconfig_playground.yaml (hunk 1 above)
Comment (achaikou) [reply radixconfig_playground.yaml:40]: No, it is not.

I look at playground config like something third-priority now. It is useful sometimes, most of the time it isn't.

So last time I updated the variables I for some reason didn't update playground file. With this new change it was of interest, as I wanted to test arm performance and you can't do that without changing the config and config is the same for all environments.
So I decided to add the missing parts, but didn't want to create a separate commit. Probably should have had.

File: radixconfig_playground.yaml (hunk 2 above)
Comment (achaikou) [reply radixconfig_playground.yaml:52]: These now are the same configs as in production/test-large.

As I was comparing performance I wanted specs to be as similar as possible now. Decided to include in the commit as it makes more sense to have expected characteristics whenever we need to recreate playground environment.
